### ISSUE TITLE
Keep getting an gpg error saying something is not installed 

### ISSUE THREAD

**Issue by cimonwoo** [desktop/desktop#18659]
### The problem

Cannot commit signed commits. Keep getting the following error about keyboxd probably not installed: 

_**gpg: error running '/usr/lib/gnupg/keyboxd': probably not installed
gpg: failed to start keyboxd '/usr/lib/gnupg/keyboxd': Configuration error
gpg: can't connect to the keyboxd: Configuration error
gpg: error opening key DB: No Keybox daemon running
gpg: skipped "7A31ED5C959237B4": Input/output error
gpg: signing failed: Input/output error
error: gpg failed to sign the data
fatal: failed to write commit object**_

### Release version

3.3.18

### Operating system

Windows 10

### Screenshots

![image](https://github.com/desktop/desktop/assets/42353196/3d1d1fb8-26a2-445b-9325-7a5dba68042a)

### Additional context

This is happening consistently after the last update. Essentially making the app useless since I have to sign all my commits.

**Comment by steveward** [comment]
@cimonwoo thanks for the issue. You should be able to resolve this by running the following command:

`gpg-connect-agent reloadagent /bye`

Even though GPG is not officially supported in GitHub Desktop [this guide](https://gist.github.com/xavierfoucrier/c156027fcc6ae23bcee1204199f177da) is a good start to verify there aren't any specific configuration issues.

We're also tracking this in https://github.com/desktop/desktop/issues/14274. I'm going to close this out, but let me know if you have any further issues.

**Comment by OussamaBenmbarek** [comment]
Hello @steveward ,
I have the same problem on Windows 11 Enterprise
I tried the command: gpg-connect-agent reloadagent /bye
but nothing works 
Version 3.4.1 (x64)
![image](https://github.com/desktop/desktop/assets/168166178/fc5a82e5-92d3-47ba-9f91-321319521282)
![image](https://github.com/desktop/desktop/assets/168166178/ff695245-85fb-4bca-a33f-d757a8bfde39)

**Comment by ArconWell** [comment]
Same thing for me...

**Comment by alexandre-machado** [comment]
I have the same problem even after running the command `gpg-connect-agent reloadagent /bye`

**Comment by johncheng2022** [comment]
I have the same problem even after running the command gpg-connect-agent reloadagent /bye

**Comment by p10tyr** [comment]
I have the same problem even after running the command `gpg-connect-agent reloadagent /bye`

**Comment by p10tyr** [comment]
So what I did to resolve the problem was to make sure my key was added in kleopatra.

1. export the key in gitbash `gpg --export-secret-key -a 12...XYZ > private.asc`
2. import the file into kleopatra using import button
3. made sure that git was using gnu gpg (double check your paths)
`where gpg` - in gitbash will tell you where you have gpg binaries
` git config --global gpg.program "C:\Users\Zef\AppData\Local\Programs\GnuPG\bin\gpg.exe"`


ref: https://blog.bsk.im/2018/01/18/signing-git-commits-with-gpg-on-windows-feat-sourcetree/
